e2e/app/element/base-element.ts

```diff
@@ -149,25 +149,25 @@ export default class BaseElement extends Container {
 
   /**
    * Clear existing value in textbox then type new text value.
-   * @param {string} The texts string.
+   * @param textValue The text string.
    * @param options The typing options.
    */
   async type(textValue: string, options?: { delay: number }): Promise<this> {
 
-    const clearAndType =  async (txt: string, opts?: { delay: number }): Promise<string> => {
+    const clearAndType = async (txt: string, opts?: { delay: number }): Promise<string> => {
       await this.clear();
       await this.asElementHandle().then((handle: ElementHandle) => handle.type(txt, opts));
       return this.getProperty<string>('value');
     }
 
     let maxRetries = 1;
     const typeAndCheck = async () => {
-      const actualValue = (await clearAndType(textValue, options)).trim();
```

Comment:
This compared a trimmed string to an untrimmed string.  So: it always failed when passed an untrimmed string